site/docs-md/guides/push-notifications-firebase.md

```diff
@@ -0,0 +1,369 @@
+
+We need to modify the App target pods to include Firebase. To do that, add `pod Firebase/Messaging` to your `target 'App'` section, like so:
+
+```ruby
+import Capacitor
+import Firebase # Add this line
+
+...
+
+target 'App' do
+capacitor_pods
+# Add your Pods here
+pod 'Firebase/Messaging' # Add this line
+end
+```
+
+Your `Podfile` should look something like this:
+
+```ruby
+platform :ios, '11.0'
+use_frameworks!
+
+# workaround to avoid Xcode 10 caching of Pods that requires
+# Product -> Clean Build Folder after new Cordova plugins installed
+# Requires CocoaPods 1.6 or newer
+install! 'cocoapods', :disable_input_output_paths => true
+
+def capacitor_pods
+  # Automatic Capacitor Pod dependencies, do not delete
+  pod 'Capacitor', :path => '../../node_modules/@capacitor/ios'
+  pod 'CapacitorCordova', :path => '../../node_modules/@capacitor/ios'
+  
+  # Do not delete
+end
+
+target 'App' do
+  capacitor_pods
+  # Add your Pods here
+  pod 'Firebase/Messaging'
+end
+```
+
+### Add Initialization Code
+
+To connect to Firebase when your iOS app starts up, you need to add the following initialization code to your `AppDelegate.swift` file, in the `application(didFinishLaunchingWithOptions)` method:
+
+```swift
+FirebaseApp.configure()
+```
+
+Your `AppDelegate.swift` file's `application(didFinishLaunching)` method should look something like this:
+
+```swift
+var window: UIWindow?
+
+func application(_ application: UIApplication,
+  didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+  FirebaseApp.configure()
+  return true
+}
+```
+
+### Update the Project
+
+Now we'll need to ensure that our iOS project is updated with the proper Firebase CocoaPod installed.
+
+*Note*: This part can take awhile as CocoaPods needs to download all the appropriate files/dependencies.
+
+```bash
+npx cap update ios
+```
+
+### Upload the APNS Certificate or Key to Firebase
+
+If you followed the instructions from the beginning, you'll have created an Apple APNS Certificate or an APNS Auth Key in the Apple Developer portal. You need to upload one of these to Firebase before Firebase can talk to APNS and send push notifications to your application.
+
+To upload your certificate or auth key, from the **Project Overview** page:
+
+1. Click on your iOS application and then the **Settings** gear icon.
+2. On the Settings page, click on the **Cloud Messaging** tab.
+3. Under the **iOS app configuration** header, upload your Auth Key or Certificate(s) using the provided **Upload** button.
+
+## Sending a Test Notification
+
+Now for the fun part - let's verify that push notifications from Firebase are working on Android and iOS!
+
+We need to fire up our application on Android or iOS to see our code on `home.page.ts` receive the notifications.
+
+To open your Android project in Android Studio:
+```bash
+npx cap open android
+```
+
+To open your iOS project in Xcode:
```

Comment:
Also good to tell the users to open the project before doing the code changes you already told them to do.